recurly/notifications/notifications.go

```diff
@@ -0,0 +1,132 @@
+package notifications
```

Comment:
Updated in [9f2146](https://github.com/blacklightcms/go-recurly/pull/6/commits/ded5ee95deb68ddc6194b97100ac475f579f2146)